balboa-http/src/main/scala/com/socrata/balboa/server/rest/MetricsRest.scala

```diff
@@ -143,8 +111,7 @@ object MetricsRest {
   }
 
   // ok, so this isn't actually correct.  Screw correct, I just want this to work.
```

Comment:
Nope. Seeing as we're planning to deprecate the protobuf support, this code will probably disappear soon.